react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import styles from './styles.module.scss';
 
-const Square = () => (
-  <button type="button" className={styles.square}>
-    {/* TODO */}
-  </button>
-);
+function Square({ onClick, value }) {
+  return (
+    <button type="button" className={styles.square} onClick={value ? undefined : onClick}>
```

Comment:
I'd leave the `onClick` as it was and add the if condition in the `handleValue` of `Game`.

Your previous version was something like this
```js
...
if(squares[i]) {
   return null;
}
...
```
The return wasn't the best option but i think the if was ok, i'd just use it the negated way
```js
if (!squares[i]) {
   ...
}
```
What do you think?